lib/server_sent_event.ex

```diff
@@ -1,10 +1,12 @@
 defmodule ServerSentEvent do
+  alias __MODULE__, as: This
```

Comment:
Why do this? is it a common thing? It looks weird to me

Reply:
We talked about it, so just for posterity: it's a convention I learned elsewhere and I like sticking to in my personal projects, it shortens and simplifies all the `%__MODULE__{}` bits in your code